typescript/agentkit/src/action-providers/spl/README.md

```diff
@@ -0,0 +1,30 @@
+# SPL Action Provider
+
+This directory contains the SPL action provider implementation, which provides SPL token transfer functionality.
```

Comment:
```suggestion
This directory contains the SPL action provider implementation, which provides actions to interact with SPL Token programs.
```

please also hyperlink to the Solana official SPL Token docs https://spl.solana.com/token